epub34/rs/index.html

```diff
@@ -1205,22 +1205,28 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
-							user interface reasons). If it does, it SHOULD preserve as much as possible the cascade model of CSS.
-							Reading system developers are expected to also publicly document the user agent style sheets and how they
-							interact with EPUB creator's style settings.</p>
-					</li>
-					<li>
-						<p id="confreq-css-user-styles">
-							It MAY provide the user with an affordance that reverts the display to the author's stylesheet.</p>
+						<p id="confreq-css-rs-html-default">SHOULD support the [[html]] <a data-cite="html#rendering"
+								>suggested default rendering</a> in their user agent style sheet(s).</p>
 					</li>
 				</ul>
 
-				<p id="confreq-css-rs-html-default">In addition to supporting CSS properties as defined above, a reading
-					system's user agent style sheet SHOULD support the [[html]] <a data-cite="html#rendering">suggested
-						default rendering</a>.</p>
+				<div class="note">
+					<p>Reading system developers are expected to publicly document their user agent style sheet(s) and
+						how they interact with EPUB creator's style settings. </p>
```

Comment:
Just wondering. The user agent style sheet has, I presume, several parts: some general styling that are always active (e.g., related to pagination), and those aspects that are related to user interface choices. E.g., when I change to sepia, then certain CSS properties related to background color change. Would it be possible for RS to provide a documentation that make those type of adaptations also public? That might help authors to adapt their content...